src/builder-utils.h

```diff
@@ -216,6 +216,10 @@ gboolean flatpak_version_check (int major,
                                 int minor,
                                 int micro);
 
+gboolean appstream_version_check (int major,
```

Comment:
The name doesn't tell me what it checks. Does it check that it is at least the version specified?

Reply:
Yea, if the run time appstream version >= (maj, min, micro) that we asked for.

Reply:
i can change it to appstream_has_version the name was a copy from flatpak_version_check above.